backend/src/main/kotlin/outcobra/server/model/mapper/BaseMapper.kt

```diff
@@ -15,20 +15,24 @@ import kotlin.reflect.KClass
  * @since 1.1.0
  */
 @Component
-open class BaseMapper {
+class BaseMapper {
     @Inject
     lateinit var repositoryLocator: RepositoryLocator
     @Inject
     lateinit var userService: UserService
 
     fun validateChildren(children: List<Long>, childType: KClass<out ParentLinked>,
                          parentId: Long, parentType: KClass<out ParentLinked>) {
+        val existingChildren = children.filter { it != 0L }
+        if (parentId == 0L && existingChildren.isEmpty()) {
+            return //this parent object and all of its children are new
```

Comment:
Any specific reason you used `.not()` here? Otherwise it's a bit inconsistent and I'd prefer the usual `!`.